.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,26 @@
+name: build-and-push-feed-discovery
+
+on:
+  push:
+    branches:
+      - 'issue#1743'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Let's also include `buildx`, which gives us some benefits when building multi-stage builds:

```
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1
```

See https://docs.docker.com/buildx/working-with-buildx/ for more info.